docs/developer/PROW_CI.md

```diff
@@ -64,12 +64,278 @@ The jobs run can be seen in PRs and in the links in the README.md file of OADP r
 
 OADP operator master branch is tested against the last 3 minor OCP releases. To Update an OCP version the project is tested against, see [Update OCP version](#update-ocp-version).
 
-### TODO Creating new release branch
+### Creating new release branches
 
-https://github.com/openshift/oadp-operator/issues/1227
+To create new OADP release branches (they must follow the pattern `oadp-major.minor`, [example](https://github.com/openshift/oadp-operator/tree/oadp-1.4)):
+- First, create CI files for each new repository branch.
```

Comment:
What CI Files? Is there a link or information what the`CI files` are ?

Reply:
yes, in this same file, there are links to each OADP repo CI files

example https://github.com/openshift/oadp-operator/blob/master/docs/developer/PROW_CI.md?plain=1#L5-L8